### PR TITLE
Issue #1622 Review "Adding a User and Project" docs for grammar and capitalization

### PR DESCRIPTION
**Pull request by rghamilton3**
Made necessary changes and reviews.

### CODE REVIEW

File: docs/tutorials/add_user_project.rst [hunk 1]
```diff
@@ -5,58 +5,60 @@ Documentation: Adding a User and Project
...
-
-- Log on to OpenHatch website: http://openhatch.org/
-- On the Front page you will see a log in/sign up link in the menu.
-- Click on the link. You will be directed to a login page.
-- OpenHatch saves your time by allowing you to use your existing 
-  account from any one of the networking sites.
-- Choose an account from which you want to sign up.
-- Once the verification process is done for a valid account you are redirected to register page. 
-  Type in your username and click on ok.
+the OpenHatch community welcomes its users to actively participate in various
```
Comment (phildini) [inline docs/tutorials/add_user_project.rst]: I think we'd prefer not to start with a lowercase letter.

**Comment by phildini** [comment]
Other than comment above, looks good!

**Comment by willingc** [comment]
@rghamilton3 Thanks for making the change. I'm happy to merge this PR. :penguin: :sunny:

Just a tip for the future, it's helpful as a developer to create a feature branch locally and develop on it instead of master. One big advantage of this is that it will allow you to squash your commits (in this case you had 5 commits) into 1 final commit for the pull request. This lets the maintainer easily merge your pull request and keeps the project history streamlined for everyone. 

Thanks again for contributing to OpenHatch and I hope to see more pull requests from you in the future. Please let me know if you have any questions. You're doing nce work :D

**Comment by rghamilton3** [comment]
@willingc Thanks for the advice! That makes a lot of sense and I'll definitely implement that.

**Pull request merged**